javascript/ql/test/library-tests/TaintTracking/destruct.js

```diff
@@ -0,0 +1,18 @@
+function test() {
+  
+  function f(obj) {
+    let { x: { y: { z } } } = obj;
+    sink(z); // NOT OK
```

Comment:
Could we have another test to show/document that this flow only happens for taint tracking, not for normal data flow?

Reply:
Added a copy of this test that uses `DataFlow::Configuration`.